include/gazebo_gps_plugin.h

```diff
@@ -106,10 +106,10 @@ class GAZEBO_VISIBLE GpsPlugin : public SensorPlugin
   //double lon_home_ = kDefaultHomeLongitude;
   //double alt_home_ = kDefaultHomeAltitude;
 
-  // American Robotics Office
-  double lat_home_ = 42.33138 * M_PI / 180.0;  // rad
-  double lon_home_ = -71.54541 * M_PI / 180.0;   // rad
-  double alt_home_ = 113.1;                     // meters
+  // Kansas site
```

Comment:
Can we leave the AR office in here, at least in the comments? Have you tried setting the environment variables mentioned above? Maybe we don't need to hardcode the Kansas location here?

Reply:
Yes, I actually returned it to AR base because it seems like moving it to kansas might be a challenge according to Alex.